addons/account/models/account_reconcile_model.py

```diff
@@ -350,7 +350,8 @@ def _get_write_off_move_lines_dict(self, st_line, residual_balance):
                 if match:
                     sign = 1 if residual_balance > 0.0 else -1
                     try:
-                        extracted_balance = float(re.sub(r'\D' + self.decimal_separator, '', match.group(1)).replace(self.decimal_separator, '.'))
+                        extracted_match_group = re.sub(r'[^\d' + self.decimal_separator + ']', '', match.group(1))
+                        extracted_balance = float(re.sub(r'\D' + self.decimal_separator, '', extracted_match_group).replace(self.decimal_separator, '.'))
```

Comment:
```suggestion
                        extracted_match_group = re.sub(r'[^\d' + self.decimal_separator + ']', '', match.group(1))
                        extracted_balance = float(extracted_match_group.replace(self.decimal_separator, '.'))
```
Don't you think the second regex `r'\D' + self.decimal_separator` is redundant?
Since the `decimal_separator` will already be removed by `re.sub(r'[^\d' + self.decimal_separator + ']', '', match.group(1))`.
I don't think there could be an occurrence of both a non-digit and the `decimal_separator` consecutively, right ?

Reply:
Hello, thanks for the review!

Yeah, totally :)

I made the change :)